modules/ROOT/nav.adoc

```diff
@@ -222,18 +222,6 @@
 ** xref:reference:rpk/index.adoc[rpk Commands]
 *** xref:reference:rpk/rpk-commands.adoc[]
 *** xref:reference:rpk/rpk-x-options.adoc[rpk -X]
-*** xref:reference:rpk/rpk-cloud/rpk-cloud.adoc[rpk cloud]
-**** xref:reference:rpk/rpk-cloud/rpk-cloud-auth.adoc[]
-***** xref:reference:rpk/rpk-cloud/rpk-cloud-auth-delete.adoc[]
-***** xref:reference:rpk/rpk-cloud/rpk-cloud-auth-list.adoc[]
-***** xref:reference:rpk/rpk-cloud/rpk-cloud-auth-use.adoc[]
-**** xref:reference:rpk/rpk-cloud/rpk-cloud-byoc.adoc[]
-***** xref:reference:rpk/rpk-cloud/rpk-cloud-byoc-install.adoc[]
-***** xref:reference:rpk/rpk-cloud/rpk-cloud-byoc-uninstall.adoc[]
-**** xref:reference:rpk/rpk-cloud/rpk-cloud-cluster.adoc[]
-***** xref:reference:rpk/rpk-cloud/rpk-cloud-cluster-select.adoc[]
-**** xref:reference:rpk/rpk-cloud/rpk-cloud-login.adoc[]
-**** xref:reference:rpk/rpk-cloud/rpk-cloud-logout.adoc[]
```

Comment:
We should also remove `*** xref:api:ROOT:cloud-api.adoc[]` from the nav tree?

Reply:
done